pr_agent/settings/configuration.toml

```diff
@@ -1,6 +1,6 @@
 [config]
-model="gpt-4" # "gpt-4-0125-preview"
-model_turbo="gpt-4-0125-preview"
+model="gpt-4" # "gpt-4-turbo-preview"
+model_turbo="gpt-4-turbo-preview"
```

Comment:
Definitely not this.

the **default** model should be fixed.

I don't want the default model to be dangling, and changing without proper checks

Reply:
any user can change this on his local deployment, if he wishes

Reply:
Thank you for bringing this to my attention. I completely agree with your point.